### PR TITLE
test feature

### PR DESCRIPTION
**Pull request by Hoa-Huong**
Screenshot from 2020-01-03 15-40-23
==> Title:test feature

==> Redmine: 

==> Attachment File:

### CODE REVIEW

**Comment by Hoa-Huong** [comment]
ready

File: spec/features/create_author_spec.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.feature "Create Author", type: :feature do
+  let!(:user){FactoryBot.create :user, role: Settings.enum_admin}
+
+  before do
+    visit login_path
+    fill_in "session[email]", :with => user.email
+    fill_in "session[password]", :with => user.password
```
Comment (manhbnt-0960) [inline spec/features/create_author_spec.rb]: thay => đi e

File: spec/features/create_author_spec.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    scenario "Admin creates a new author" do
+      fill_in "author[name]", with: Faker::Lorem.characters(number: Settings.name_author)
+      fill_in "author[description]", with: Faker::Lorem.characters(number: Settings.description_author)
+      click_button "commit"
+      expect(page).to have_text I18n.t("success")
+      expect(page).to have_current_path("/en/admin/authors")
+    end
+  end
+
+  feature "create success" do
```
Comment (manhbnt-0960) [inline spec/features/create_author_spec.rb]: 2 feature succes à e

**Comment by Hoa-Huong** [comment]
ready

**Pull request merged**